index.js

```diff
@@ -1,268 +0,0 @@
-    compPick === "Yes" ||
-    compPick === "YES" ||
-    compPick === "yes"
-  ) {
-    console.log("I got it you little shit.");
-    guessRightLoop();
-  } else {
-    console.log(
-      "Hey dingus, please write either 'Yes' or 'No'. I'm a computer, make it easy for me.\n"
-    );
-    compPickLoop();
-  }
-}
-
-//Wrong guess loop
-async function guessWrongLoop() {
-  let secondPick = await ask("Should I guess higher or lower?\n");
-  compPick = "";
-  if (
-    secondPick === "higher" ||
-    secondPick === "Higher" ||
-    secondPick === "h" ||
-    secondPick === "H"
-  ) {
-    minRange = numGuess;
-    
-
-
-    compPickLoop();
-    //loopCheckRange();
-  } else if (
-    secondPick === "lower" ||
-    secondPick === "Lower" ||
-    secondPick === "l" ||
-    secondPick === "L"
-  ) {
-    maxRange = numGuess;
-    compPickLoop();
-    //loopCheckRange;
-  } else {
-    console.log(
-      "This means nothing to me. Please use either 'higher' or 'lower', 'h', or 'l'.\n"
-    );
-    guessWrongLoop();
-  }
-}
-
-
-//Right Guess Loop - start again or exit//
-async function guessRightLoop() {
-
-  let startAgain = await ask("Would you like to play again?\n");
-  if (
-    startAgain === "y" ||
-    startAgain === "Yes" ||
-    startAgain === "YES" ||
-    startAgain === "yes"
-  ) {
-    gamePick();
-  } else if (
-    startAgain === "n" ||
-    startAgain === "No" ||
-    startAgain === "NO" ||
-    startAgain === "no"
-  ) {
-    console.log("Okay byeeee!");
-    process.exit();
-  }
-}
-
-
-
-//Work in Progress - not quite working how I want it too//
-//Loop check range//
-function loopCheckRange () {
-  if (minRange < maxRange) {
-    compPickLoop();
-        //console.log check block start//
-        console.log("minRange is " + minRange);
-        console.log("maxRange is " + maxRange);
-        //console.log check block end//
-    
-  } if (minRange === maxRange){
-    console.log("We've come to the end of the road here. Both your ranges seem to be " + maxRange +". I have no choice but to declare myself winner. Suck it.")
-    guessRightLoop();
-  } else if (minRange > maxRange) {
-    console.log("It looks like your range makes no sense. Lets try it again\n");
-    guessWrongLoop();
-}
-}
-
-
-
-//2nd part of the game - You try and guess the computer's number
-let playerMinRange;
-let playerMaxRange;
-let compNum;
-let compNumInt;
-
-async function playerStart() {
-  console.log(
-    "I'm going to guess a number. Let's figure out how hard you want to make it"
-  );
-  playerMaxRange = await ask("What's the highest number I can set?\n");
-  playerMinRange = await ask("What's the lowest number I can set?\n");
-  compNum = pickNum(playerMinRange, playerMaxRange);
-  compNumInt = Number(compNum);
-  console.log("Computer picks " + compNumInt);
-  playerPick();
-}
-
-//Random number picker within range (inclusive) //
-function pickNum(min, max) {
-  let range = max - min + 1;
-  let randInt = min + Math.floor(Math.random() * range);
```

Comment:
I was able to get this portion of the game 100% working by making a minor change to line 224. Changing the equation to `let randInt = Math.floor(Math.random() * range + min)` will resolve this bug. It appears to be just a order of operations thing. 

If you run the game in it's current state but add `console.log(compNum)` between lines 215-216 you can see in the terminal that a very high number gets returned after the `pickNum()` function is called on the users range.